package.json

```diff
@@ -19,6 +19,7 @@
     "css-loader": "^3.5.3",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "fs": "0.0.1-security",

```

Comment:
You DO NOT need this. It's there as a security placeholder. FS is built into Nodejs